File: sykepengesoknad/src/main/java/no/nav/syfo/kafka/sykepengesoknad/dto/SoknadsperiodeDTO.java
package no.nav.syfo.kafka.sykepengesoknad.dto;

import lombok.Builder;
import lombok.Value;

import java.time.LocalDate;

@Value
@Builder(toBuilder = true)
public class SoknadsperiodeDTO {
    LocalDate fom;
    LocalDate tom;
    Integer sykmeldingsgrad;
    Integer faktiskGrad;
    Double avtaltTimer;
    Double faktiskTimer;
    SykmeldingstypeDTO sykmeldingstype;
}

File: sykepengesoknadbehandlingsdager/src/main/kotlin/no/nav/syfo/kafka/sykepengesoknadbehandlingsdager/dto/SoknadsperiodeDTO.kt
package no.nav.syfo.kafka.sykepengesoknadbehandlingsdager.dto

import java.time.LocalDate

data class SoknadsperiodeDTO(
        val fom: LocalDate? = null,
        val tom: LocalDate? = null,
        val sykmeldingsgrad: Int? = null,
        val faktiskGrad: Int? = null,
        val avtaltTimer: Double? = null,
        val faktiskTimer: Double? = null,
        val sykmeldingstype: SykmeldingstypeDTO? = null
)
